docs/source/projections/gall.rst

```diff
@@ -4,7 +4,74 @@
 Gall (Gall Stereographic)
 ********************************************************************************
 
+The Gall stereographic projection, presented by James Gall in 1855, is a cylindrical projection. It is neither equal-area nor conformal but instead tries to balance the distortion inherent in any projection.
+
++---------------------+--------------------------------------------------------------------------------+
+| **Classification**  | Transverse and oblique cylindrical                                             |
++---------------------+--------------------------------------------------------------------------------+
+| **Available forms** | Forward and inverse, Spherical                                                 |
++---------------------+--------------------------------------------------------------------------------+
+| **Defined area**    | Global                                                                         |
++---------------------+--------------------------------------------------------------------------------+
+| **Implemented by**  | Gerald I. Evenden                                                              |
++---------------------+--------------------------------------------------------------------------------+
+| **Options**                                                                                          |
++---------------------+--------------------------------------------------------------------------------+
```

Comment:
On the calcofi page I wrote "No special options for this projection" under options. I think we should do that for all projections with no extra options. Otherwise the table looks a bit unfinished.